src/compositor/html/html-transforms.ts

```diff
@@ -187,6 +187,7 @@ export const init = (
           onEvent: compositor.on,
           onNewSource: (cb) => _onNewSourceHandlers.push(cb),
           onUpdate: (cb) => _onUpdateHandlers.push(cb),
+          nodeId : node.id
```

Comment:
Node Id added to transform context. This make nodeId available in every **create** function context element.